### ISSUE TITLE
ipynb file with R kernel

### ISSUE THREAD

**Issue by dbose** [airbnb/knowledge-repo#358]
Auto-reviewers: @NiharikaRay @matthewwardrop @earthmancash @danfrankj

Having difficulty in adding a ipynb (R-kernel) file to the KP. The postprocessor fails at 
>> "ValueError: YAML header is missing. Please ensure that the top of your post has a header of the following form:" 

Even though I have following header

```
---
title: "Scroll depth analysis"
authors:
- dbose 
tags:
- ab_tests
- experiments
created_at: 2017-11-20
updated_at: 2017-11-20
tldr: Test
---
```

**Comment by shashj199** [comment]
@dbose Remove the quotes from Scroll depth analysis. It should work

---
title: Scroll depth analysis
authors:
- dbose 
tags:
- ab_tests
- experiments
created_at: 2017-11-20
updated_at: 2017-11-20
tldr: Test
---
